Multimodal-image-audio-best-practices-2026.md

```
@@ -0,0 +1,799 @@
+- Gemini 3.1 Flash Live: Making audio AI more natural and reliable（Google公式）: https://blog.google/innovation-and-ai/models-and-research/gemini-models/gemini-3-1-flash-live/
+- How to use Gemini Live API Native Audio in Vertex AI（Google Cloud Blog）: https://cloud.google.com/blog/topics/developers-practitioners/how-to-use-gemini-live-api-native-audio-in-vertex-ai
+- Gemini 3.1 Flash Live vs GPT Realtime 1.5比較: https://flowtivity.ai/blog/gemini-3-1-flash-live-vs-gpt-realtime-1-5-voice-agent-comparison-2026/
+- GPT-Realtime-2 vs Gemini Live API比較: https://webscraft.org/blog/gptrealtime2-vs-gemini-live-api-scho-obrati-dlya-golosovogo-agenta-u-2026-rotsi?lang=en
+- Gemini 3.1 Flash Live: Google's Realtime Voice AI（Safina AI）: https://safina.ai/en/blog/gemini-3-1-flash-live-realtime-voice-ai/
+- Google Releases Gemini 3.1 Flash Live（MarkTechPost）: https://www.marktechpost.com/2026/03/26/google-releases-gemini-3-1-flash-live-a-real-time-multimodal-voice-model-for-low-latency-audio-video-and-tool-use-for-ai-agents/
+- Gemini Live API plugin（LiveKit Documentation）: https://docs.livekit.io/agents/models/realtime/plugins/gemini/
+- Real-Time vs Turn-Based Voice Agents in 2026: https://softcery.com/lab/ai-voice-agents-real-time-vs-turn-based-tts-stt-architecture
+
+---
+
+*本ガイドは2026年7月10日時点でWeb検索により収集した公開情報にもとづき作成されています。生成AI分野は変化が非常に速いため、実装・契約の際は必ず各社公式ドキュメントの最新版をご確認ください。*
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)